CMSgov/onemac
# OY3 - 13119 Role request removal is not working

Story: https://qmacbis.atlassian.net/browse/OY2-13119
Endpoint: https://d2akrubpzmbd2w.cloudfront.net

### Details
A bug was found showing that the self-revoke functionality in the User Page was not working.
Testing determined that the revoke was working, but the confirmation dialog was not giving up control appropriately.

### Changes
- added a call to closeConfirmation() at end of revoke funciton
- added an aria-label to the alert bar so it could be unit tested
- added tests to improve test coverage
- removed service worker from application - we have had at least one user call help desk because of caching issues and we do not use the service worker specifically for anything.

### Implementation Notes
- Confirmation Dialog replaces the browser default confirmation window.... however, the functionality is not exactly the same and that is what caused this bug.  Some thought should be put into possible tech debt next steps.

### Test Plan
1. Login as a state submitter with at least one active or pending state.
2. Navigate to User Profile page
3. Click on the "X" to self-revoke access to a state
4. Click "Confirm" to confirm the revoke action.
5. Note that the confirmation window has gone and the user is now revoked from that state.
